src/Pages/URLShortener/URLShortener.js

```diff
@@ -0,0 +1,158 @@
+import React, { useEffect, useState } from 'react';
+
+import Header from '../../Components/Header/Header';
+import './URLShortener-page.css';
+import { getAllURLs, createURL, deleteURL } from '../../APIFunctions/Cleezy';
+import { Container, Button, Row, Col, Input } from 'reactstrap';
+import { trashcanSymbol } from '../Overview/SVG';
+
+export default function URLShortenerPage(props) {
+  const [URL, setURL] = useState();
+  const [invalidURL, setInvalidURL] = useState();
+  const [alias, setAlias] = useState();
+  const [allURLs, setAllURLs] = useState([]);
+  const [loading, setLoading] = useState(true);
+  const [error, setError] = useState();
+  const [aliasTaken, setAliasTaken] = useState();
+  const [createURLResponse, setCreateURLResponse] = useState({});
+
+  async function getURLsFromDB() {
+    const URLsFromDB = await getAllURLs(props.user.token);
+    if (!URLsFromDB.error) {
+      setAllURLs(URLsFromDB.responseData);
+      setLoading(false);
+    } else {
+      setError(URLsFromDB.responseData);
+    }
+  }
+
+  async function handleCreateURL() {
+    const response = await createURL(URL, alias, props.user.token);
+    console.debug(response);
+    if (!response.error) {
+      setCreateURLResponse(response.responseData);
+      setAllURLs([response.responseData, ...allURLs]);
+      setAliasTaken(false);
+      setURL('');
+      setAlias('');
+      document.getElementById('url-box').value = '';
+      document.getElementById('alias-box').value = '';
+      return true;
+    } else if(response.error === 409) {
+      setAliasTaken(true);
+      return false;
+    } else {
+      return false;
+    }
+  }
+
+  async function checkValidURL(URL) {
```

Comment:
i link this naming more
```suggestion
  async function maybeSubmitUrl(URL) {
```